src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.cs

```diff
@@ -1340,6 +1341,100 @@ public bool WaitForExit(int milliseconds)
             return exited;
         }
 
+        /// <summary>
+        /// Instructs the Process component to wait for the associated process to exit, or
+        /// for the <paramref name="cancellationToken"/> to be canceled.
+        /// </summary>
+        /// <remarks>
+        /// Calling this method will set <see cref="EnableRaisingEvents"/> to true.
```

Comment:
Thank you for documenting this new public API. Here's a small suggestion:
```suggestion
        /// Calling this method will set <see cref="EnableRaisingEvents"/> to <see langword="true" />.
```

Reply:
Thanks! Fixed!